### ISSUE TITLE
Integration with `pyam` for visualization and data analysis

### ISSUE THREAD

**Issue by danielhuppmann** [sentinel-energy/friendly_data#2]
I want to express my interest in developing `to_sentinel()` and `read_sentinel()` functions in the [pyam](https://pyam-iamc.readthedocs.io) package, so that one can use the sentinel-archive together with the plotting and analysis features of pyam.

We already have integration with `datapackage-py` ([read the docs](https://pyam-iamc.readthedocs.io/en/stable/api/io.html#the-frictionless-data-package)), so this should be fairly straightforward.

Please ping when this would be ready to be implemented.

**Comment by suvayu** [comment]
@danielhuppmann I don't know how I missed this issue, extremely sorry.  I'll definitely sync with you when we are at that stage.  Maybe in a few weeks.  Right now I'm focused on some of the basic features needed for analysis, and building a beginner friendly [web interface](https://github.com/sentinel-energy/sentinel-archive-ui) to create these data packages.

**Comment by suvayu** [comment]
@danielhuppmann I am closing this issue as I implemented a scheme to convert a Friendly data package into IAMC.  You can review the feature set in the [docs](https://sentinel-energy.github.io/friendly_data/iamc-conversion.html).

It's not entirely automatic, as the mapping between a model's internal name to an IAMC hierarchy (like the openEntrance/nomenclature) is still a manual process.  That said, once someone has done that work for their model, converting their Friendly data package to IAMC can be easily automated.  Updating/maintaining the mapping should also be much simpler.

In conclusion, `pyam` doesn't need to implement anything to take advantage.  Further feedback is always welcome :)

**Comment by danielhuppmann** [comment]
Thanks, @suvayu - had a look at the documentation page, looks great! I hope to take a look soon at the implementation and a simple example and will provide feedback...
